### PR TITLE
Fluent Theme: Update Toggle styles

### PR DESCRIPTION
**Pull request by mikewheaton**
#### Pull request checklist

- [x] Addresses an existing issue: #6712
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updates Toggle styles to match the latest design.

### CODE REVIEW

File: packages/fluent-theme/src/fluent/styles/Toggle.styles.ts [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
+        height: '20px',
+        borderRadius: '10px',
+        padding: '0 4px'
+      },
+      !disabled && [
+        checked && {
+          selectors: {
+            ':hover': [
+              {
+                backgroundColor: CommunicationColors.shade20
```
Comment (mikewheaton) [inline packages/fluent-theme/src/fluent/styles/Toggle.styles.ts]: The `palette` object should be used wherever possible. See if this color and the two below exist in that object before using the Fluent palettes.

File: packages/fluent-theme/src/fluent/styles/Toggle.styles.ts [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
-    height: '20px',
-    borderRadius: '10px',
-    padding: '0 4px'
-  },
-  thumb: {
-    width: '12px',
-    height: '12px',
-    borderRadius: '12px',
-    borderColor: 'transparent'
+import { IToggleProps } from '../../../../office-ui-fabric-react/lib';
```
Comment (Vitalius1) [inline packages/fluent-theme/src/fluent/styles/Toggle.styles.ts]: this import should be `import { IToggleProps } from 'office-ui-fabric-react/lib/Toggle'`... I believe you used the auto import feature which is sometimes buggy))

Comment (mikewheaton) [reply packages/fluent-theme/src/fluent/styles/Toggle.styles.ts]: Thanks for catching that, I'll update it. I love the auto import feature... 😢

**Comment by Vitalius1** [comment]
@mikewheaton I hope you don't mind that I did a change directly to your branch changing the imports))

**Comment by mikewheaton** [comment]
@Vitalius1 Not at all, thanks for catching that messy import!

**Pull request merged**